ui/app/components/meta-table/index.ts

```diff
@@ -0,0 +1,92 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import ApiService from 'waypoint/services/api';
+import FlashMessagesService from 'waypoint/services/pds-flash-messages';
+import {
+  Deployment,
+  Ref,
+  ExpediteStatusReportRequest,
+  StatusReport,
+  GetJobStreamRequest,
+  GetJobStreamResponse,
+  Job,
+} from 'waypoint-pb';
+
+interface OverviewTableArgs {
```

Comment:
I quite like to call these just `Args`, that way I don’t need to rename the type if I rename the component.